### ISSUE TITLE
problem when insert codeblocks use ctrl+m ctrl+c

### ISSUE THREAD

**Issue by NalanHan** [mdickin/vscode-markdown-shortcuts#45]
Hi,
when I insert code blocks with `ctrl+m ctrl+c ` in markdown,it is like this:
<pre> ```autoauto```</pre>
How can I change the default behavior to 
<pre>
```
somecode
```
</pre>
Thanks.

**Comment by NalanHan** [comment]
Hi,
when I insert code blocks with `ctrl+m ctrl+c ` in markdown,it is like this:
```autoauto```
How can I change the default behaviour to 
```
```
Thakns.

**Comment by mdickin** [comment]
Hi NalanHan,

Thanks for reporting this. This looks like a bug in the extension due to a change in VSCode. The extension is using the "EOL" setting to determine what end of line format to use. At some point, the "auto" option was added, which is why you're seeing that.

A workaround for this is to change your EOL setting to either `\n` or `\r\n`, instead of "auto".

**Comment by mdickin** [comment]
To change this setting, go to File > Preferences > Settings.

![image](https://user-images.githubusercontent.com/16612667/51540424-05fe9a00-1e1c-11e9-9c1d-a2da9c6ab609.png)

**Comment by NalanHan** [comment]
Hi mdickin，
Thanks for the reply. It solves the problems perfectly.

**Comment by jmooring** [comment]
Please re-open this issue. Setting the eol to \n or \r\n is a workaround, not a solution. It took me a couple of hours to discover the workaround.

**Comment by mdickin** [comment]
Agreed, this needs a proper fix. I should be pushing out an update shortly

**Comment by jmooring** [comment]
Looks like we can simply replace 
```
var newLine = vscode.workspace.getConfiguration('files').get('eol', '\r\n');
```
with
```
var newLine = require('os').EOL;
```

**Comment by jmooring** [comment]
My last comment is pointless given your last commit. Thank you for fixing this so rapidly.

**Comment by mdickin** [comment]
This has been resolved in v0.10.0. Thanks for the feedback!

And thank you @jmooring for looking into a fix, even if we got there at the same time :)
